x-pack/platform/plugins/shared/ml/public/application/services/ml_api_service/results.ts

```diff
@@ -240,6 +244,62 @@ export const resultsApiProvider = (httpService: HttpService) => ({
       version: '1',
     });
   },
+
+  getTopInfluencers(payload: GetTopInfluencersRequest) {
+    const body = JSON.stringify(payload);
+    return httpService.http<InfluencersByFieldResponse>({
+      path: `${ML_INTERNAL_BASE_PATH}/results/top_influencers`,
+      method: 'POST',
+      body,
+      version: '1',
+    });
+  },
+
+  getScoresByBucket(payload: {
+    jobIds: string[];
+    earliestMs: number;
+    latestMs: number;
+    intervalMs: number;
+    perPage?: number;
+    fromPage?: number;
+    swimLaneSeverity?: Array<{ min: number; max?: number }>;
+  }) {
+    const body = JSON.stringify(payload);
+    return httpService.http<{
```

Comment:
nit, this response type could be moved to a common type for `getScoresByBucket` and `getInfluencerValueMaxScoreByTime`